_dashboards/discover/index-discover.md

```diff
@@ -1,104 +1,115 @@
 
-1. In the search toolbar, enter the Boolean query. For example, enter `FlightDelay:true AND FlightDelayMin >= 60` to search the data for flights delayed by 60 minutes or more.
-1. Choose **Update**.
-1. Optional: Choose the arrow (`>`) in a table row to expand the row and view the document table details.
+To add or delete fields in a document table, follow these steps:
 
-## Filtering data
+1. View the data fields listed under **Available fields** and select the plus icon ({::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/plus-icon.png" class="inline-icon" alt="plus icon"/>{:/}) to add the desired fields to the document table. The field will be automatically added to **Selected fields** and the document table. For this example, choose the fields `Carrier`, `AvgTicketPrice`, and `Dest`.
+1. To arrange or sort the columns, select the column header containing the field name and select the desired action. An example is shown in the following image. 
 
-Filters allow you to refine sets of documents to subsets of those documents. For example, you can filter data to include or exclude certain fields, as shown in the following image.
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/arrange-columns.png" alt="Document table interface" width="700"/>
 
-![Filtering data interface]({{site.url}}{{site.baseurl}}/images/discover-filter.png)
+You can view individual or multiple fields in the document table to gain deeper knowledge about your data. To explore data in the document table, follow these steps: 
 
-Follow these steps to filter data:
+1. Choose the inspect icon ({::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/inspect-icon.png" class="inline-icon" alt="inspect icon"/>{:/}) to open the **Document Details** window.
+1. Review the data details. You can switch between the **Table** and **JSON** tabs to view the data in your preferred format. 
+1. Select **View surrounding documents** to view data about events that occurred before or after the data you're searching or select **View single document** to view data about a particular event. An example is shown in the following image.
 
-1. In the filter bar, choose **Add filter**.
-1. Select options from the **Field**, **Operator**, and **Value** dropdown lists. For example, `Cancelled`, `is`, and `true`.
-1. Choose **Save**.
-1. To remove the filter, choose the close icon (x) next to the filter name.
-1. Optional: Add more filters to further explore the data.  
+### Search your data
+
+You can use the search toolbar or enter a [DQL]({{site.url}}{{site.baseurl}}/dashboards/discover/dql/) query using the **DevTools** console to search data. While the search toolbar is best for basic queries, such as searching by a field name, DQL is best for complex queries, such as searching data using a term, string, Boolean, date, range, or nested query. DQL provides suggestions for fields and operators as you type, helping you build structures queries.
```

Comment:
"structures queries" is throwing me off. Is that actually the correct phrasing?

Reply:
Should have been "structured". Change made.